lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -76,29 +76,7 @@ def define
           if response == "none"
             outcome :outcome_not_affected
           else
-            case benefit_types.shift
-            when :bereavement then question :bereavement_amount?
-            when :carers then question :carers_amount?
-            when :child_benefit then question :child_benefit_amount?
-            when :child_tax then question :child_tax_amount?
-            when :esa then question :esa_amount?
-            when :guardian then question :guardian_amount?
-            when :incapacity then question :incapacity_amount?
-            when :income_support then question :income_support_amount?
-            when :jsa then question :jsa_amount?
-            when :maternity then question :maternity_amount?
-            when :sda then question :sda_amount?
-            when :widowed_mother then question :widowed_mother_amount?
-            when :widowed_parent then question :widowed_parent_amount?
-            when :widow_pension then question :widow_pension_amount?
-            when :widows_aged then question :widows_aged_amount?
-            else
-              if housing_benefit == 'yes'
-                question :housing_benefit_amount?
-              else
-                question :single_couple_lone_parent?
```

Comment:
I would've preferred to see this change (i.e. "Remove housing benefit conditional in Q4 and Q5") in a separate commit.